model/controller/dual_mode_controller.cc

```diff
@@ -1678,6 +1706,10 @@ void DualModeController::SetEventFilter(CommandView command) {
   DEBUG(id_, "   filter_type={}",
         bluetooth::hci::FilterTypeText(command_view.GetFilterType()));
 
+  if (command_view.GetFilterType() != bluetooth::hci::FilterType::CLEAR_ALL_FILTERS) {
```

Comment:
@hchataing - This particular change will not work with my specific use case. The BT stack I'm using sends SentEventFilter commands. Could you provide my clarity about why this was added and if there are other ways that we can work around this?

Reply:
Hi @akuker, thanks for reporting. I added this check in lieu of a full implementation, since the feature was apparently unused in our testing context.
I will provide a full implementation if you to need it :+1:

Reply:
I am interested to know how your are using Rootcanal by the way, that will help me anticipate similar issues in the future :smile:

Reply:
Hi @hchataing . Thank you for the quick response!

A full implementation would be nice, but not 100% required. Our simulation seemed to be OK without it. 

For our use case, we are simulating smart home / IoT devices and using netsim/rootcanal to connect them all together. Depending upon the test scenario, we spin up one or more AOSP emulator devices to test interaction between the phones and devices. For this specific issue, the smart home device is using a BT stack derived from Bluedroid. 

We started out using the Bumble link relay, but it doesn't look like there is a lot of activity in that area.

I have a number of other minor feature requests/questions about netsim. Is there an appropriate forum to ask those? Is https://issuetracker.google.com/ appropriate? (since this is just a mirror of the AOSP repo)

Reply:
Thank you for the answer !  https://issuetracker.google.com/ is a good place to post your questions.
You may use `componentid:1261108` for requests that are specific to Bluetooth emulation,
and `componentid:1203577` for more general questions (e.g. emulator integration).

Reply:
HI @hchataing  - I'm unable to see those two components. Its giving me an access denied error when I try to access them. Is there a way I can get access to file issues against these components? Or, should I file them against something more generic? (Note: my issue tracker account is akukeramzn@gmail.com)

Reply:
Hi, sorry for the delay.
These components are not public apparently. 
I will check how to change the privacy setting, meanwhile feel free to report change requests in github issues 
and I will upload them to the issue tracker

Reply:
I started working on implementing the HCI Set Event Filter command in RootCanal, but I would like to go further with the update as I won't finish it soon. I suggest an older version of RootCanal until I release the implementation.

Reply:
Thank you for the update @hchataing ! (apologies for the delay - I was offline for a couple weeks)

For now, I'm using a patched version of netsim/rootcanal with those few lines removed.

I'm going to add a couple feature requests to this github project, since I don't have access in issuetracker. Hopefully that's OK!

Reply:
I commented out the hard assert on the Event Filter type in the github version to avoid breaking your use case; it will stay this way until I get around to actually implementing the command. 

It's perfectly OK to add the feature requests to the github project :)

Reply:
I also found a work-around using Bumble's hci-bridge to short circuit that particular command. So, I should be good using the standard implementation.

Thank you again for your support!! :)